src/components/layout/RightSectionWallet.vue

```diff
@@ -21,25 +21,50 @@
         {{ address }}
       </div>
       <div class="not-bootstrap-row">
-        <AeAmount :amount="balance" />
+        <Dropdown
+          v-if="hasContractV2Address"
+          :options="tokenBalancesOptions"
+          :method="updateTokenBalance"
+          :selected="selectedTokenBalance.token"
+        >
+          <template #diplayValue>
+            <AeAmount
+              class="not-bootstrap-row"
+              :amount="selectedTokenBalance.balance"
+              :token="selectedTokenBalance.token"
```

Comment:
These values should be taken as a current dropdown value from the slot scope, no need to define `selectedTokenBalance` for that.